apps/condo/domains/acquiring/access/RecurrentPayment.js

```diff
@@ -0,0 +1,31 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema acquiring.RecurrentPayment 'status:Text;tryCount:Integer;state:Json;billingReceipts:Json'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+const userFilter = (id) => ({ recurrentPaymentContext: { serviceConsumer: { resident: { user: { id } } } } })
```

Comment:
you can move this function somewhere to lib (as you copy-pate it to multiple files)